onnxruntime/test/python/transformers/test_beam_search.py

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env python
+# coding: utf-8
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation.  All rights reserved.
+# Licensed under the MIT License.  See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+import unittest
+import os
+import pytest
+
+from parity_utilities import find_transformers_source
+if find_transformers_source():
+    from convert_beam_search import main as run
+else:
+    from onnxruntime.transformers.convert_beam_search import main as run
+
+
+class TestBeamSearch(unittest.TestCase):
```

Comment:
I believe we need test cases at operator level too, just like others.

Reply:
This is operator level test (by comparing the parity with huggingface transformer).

We can probably also add a small model and hard code the expected result, and enable test in CI pipeline later. There's a comment "#TODO: use a smaller model and enable tests in CI pipeline" in this file.

Reply:
I meant .cc